topologies/routing/files/MenuOptions.yaml

```diff
@@ -27,6 +27,33 @@ options:
   complete: 
   -   command: "/usr/local/bin/ConfigureTopology.py -t complete"
       description: "Complete all Sections (complete)"
```

Comment:
Maybe it will be good to add in a `\n` at the end of the complete description.  Would add a line break between the standard routing topology and ring topo.  So it would look like:
```
description: "Complete all Sections (complete)\n"
```

I'm open to discussion on this.

Reply:
Agreed.  The menu option formatting and the specific description text is what I was hoping to review on the call with everyone.  I definitely think incorporating your suggestion is something we should do.  I'll update it after the call on Monday.